src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -5553,6 +5555,124 @@ void M()
             Assert.True(semanticInfo.Symbol.GetSymbol().IsFromCompilation(compilation));
         }
 
+        [WorkItem(59709, "https://github.com/dotnet/roslyn/issues/59709")]
+        [Fact]
+        public void LocalForEach()
```

Comment:
>LocalForEach [](http://example.com/codeflow?start=20&length=12)

It feels like these tests do not belong in this file. We are not testing semantic model, just using it to get to the symbol

Reply:
Consider adding a new file for locals under src\Compilers\CSharp\Test\Symbol\Symbols\Source

Reply:
Right, thanks. Moved tests: f414d9e4769faa537fc00779555afcf99f343687